react-playground/src/Pages/Home.tsx

```diff
@@ -0,0 +1,9 @@
+import { FC } from "react";
+
+export interface HomeProps {}
+
+const Home: FC<HomeProps> = (props) => {
```

Comment:
Did this make you create an empty interface in order for this to work? Just curious. Perhaps yes because of the FC usage.

Reply:
It did not force me to. I was just following convention of the components I'd written in the last PR when I was passing in props. Pretty sure I can remove the interfaces and something like this:

`const Results: FC = (props) => {
  return <h1>This is the results page</h1>;
};`